### ISSUE TITLE
[Consensus] Consensus Knowledge Transfer

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/pocket#890]
## Objective

This is an umbrella ticket to capture references and discussion around knowledge transfer of the Consensus module. The primary focus is to onboard ___, identify and resolve low hanging fruit, and set up consensus so it's ready for production.

## Origin Document

* [V1 Consensus Specification](https://github.com/pokt-network/pocket-network-protocol/blob/main/consensus/README.md).
* Add link to source code

### HotPOKT

![Screenshot 2023-01-16 at 3 08 51 PM](https://user-images.githubusercontent.com/1892194/212758382-2f3c1da4-4be3-40cf-84ef-169a7c40756c.png)

The current implementation of RainTree and the P2P module lives [here](https://github.com/pokt-network/pocket/tree/main/p2p).

## Goals

- Capture the discussion & questions needed for support & knowledge transfer of the [P2P specification](https://github.com/pokt-network/pocket-network-protocol/blob/main/p2p/README.md) in the future
- Determine the missing gaps and define a path forward on HotPOKT
- Identify an owner / Point of Contact (POC) for the V1 Consensus module
- Outline & document follow-up work to build & scale the Consensus module

## Deliverable

- 1. #268

## Non-goals / Non-deliverables

- **Updating/re-writing the [P2P specification](https://github.com/pokt-network/pocket-network-protocol/blob/main/consensus/README.md)**; though this will need to be done in the future

---

**Creator**: @Olshansk 
**Co-Owners**: ???

**Comment by Olshansk** [comment]
@red-0ne Friendly reminder that if you're ever blocked on something or waiting on someone, this should capture the first 3 months of onboarding in terms of goals & deliverables.
